### PR TITLE
Change CUIT label to CUIT/DNI

### PR DESCRIPTION
**Pull request by tcordeu**
It is very common that customers have DNI and not CUIT, for example: the
customer is "Consumidor final".

### CODE REVIEW

**Comment by tcordeu** [comment]
Esto es medio un workaround que cubre la mayoría de los casos de uso. Otra opción seria recibir el tipo de documento y en base a eso generar un label, pero tambien introduciría una nueva dependencia.

File: lib/afip_bill/views/shared/_factura_header.html.erb [hunk 1]
```diff
@@ -184,8 +184,8 @@
   <div style="left:232.44px;top:169.29px" class="cls_005"><span class="cls_005">Hasta:</span><span class="cls_007"><%= Date.parse(afip_bill["fch_serv_hasta"]).strftime("%d/%m/%Y") %></span></div>
   <div style="left:362.96px;top:169.29px" class="cls_005"><span class="cls_005">Fecha de Vto. para el pago:</span></div>
   <div style="left:495.00px;top:169.29px" class="cls_007"><span class="cls_007"><%= Date.parse(afip_bill["fch_serv_hasta"]).strftime("%d/%m/%Y") %></span></div>
-  <div style="left:21.00px;top:189.59px" class="cls_002"><span class="cls_002">CUIT:</span></div>
-  <div style="left:52.00px;top:189.59px" class="cls_006"><span class="cls_006"><%= afip_bill["doc_num"] %></span></div>
+  <div style="left:21.00px;top:189.59px" class="cls_002"><span class="cls_002">CUIT/DNI:</span></div>
+  <div style="left:65.00px;top:189.59px" class="cls_006"><span class="cls_006"><%= afip_bill["doc_num"] %></span></div>
```
Comment (etagwerker) [inline lib/afip_bill/views/shared/_factura_header.html.erb:188]: @tcordeu Interesting! Couldn't you determine the label based on the value of `afip_bill["doc_num"]`? 

If it is a CUIT/CUIL value, what does that value look like? If it is a DNI what does it look like?

**Review by etagwerker — COMMENTED** [review]
@tcordeu Thanks for submitting this! 

I added one comment regarding that label. If it is impossible to distinguish between CUIT/CUIL vs. DNI, I think this PR is a good workaround. 😄

**Comment by etagwerker** [comment]
@tcordeu pudiste ver mi comentario?

**Comment by bronzdoc** [comment]
This has been stale for too long if comments are addressed I'm happy to reopen it, thanks!

**Pull request closed without merging**